### DISCUSSION TITLE
Sound isn't diegetic

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2790] [General Questions & Discussion] [answered]
I have an mp3 near an object, and I need it to appear as if the sound is coming from that object.  I know that panner mode has settings where you can control when the sound drops off as the user moves further away. However, I've tried all kinds of settings and my sound is the same volume no matter where I am in the space.  I figure decreasing the reference and max distance should help, since it's not a huge space and the sound should drop off relatively quickly as you move away.  I'm just having any luck tweaking those parameters, or others.  I know it's hard to describe without giving you the scene file, but in general I'm wondering what settings are typically successful for diegetic sound.  Also, "loop" doesn't seem to work if I have more than 1 audio file in the scene, despite loop being checked on for both of them. Is this a possible bug?

**Comment by brianpeiris** [comment, marked as answer]
Hi @hilarytsai, Is this still an issue? Can you give us more details on what device you're using when this happens?

**Comment by robinkwilson** [comment]
Marking this as answered so it gets out of our "todo" pipeline. Please @ me if you give us more details!
